app/Http/Controllers/Api/v1/User/Controllers/GenerateTokenController.php

```diff
@@ -42,13 +44,42 @@ public function __invoke(Request $request): JsonResponse
         try {
             $user = $this->userLookup->getUpdatedUser($user, $azureUser->principal_name);
         } catch (ClientExceptionInterface) {
+            $log = new Log();
+            $log->user_id = $user->id;
+            $log->severity = '5';
```

Comment:
I'd like this to be defined in an enum or something. `5` is just a magic number here

Reply:
5 is RFC-standard for a certain severity

Reply:
Sure, but I don't want to look up an RFC just to know what is going on here. Put this in an enum.